core/src/main/scala/com/softwaremill/sttp/MonadError.scala

```diff
@@ -31,6 +31,15 @@ trait MonadAsyncError[R[_]] extends MonadError[R] {
   def async[T](register: (Either[Throwable, T] => Unit) => Unit): R[T]
 }
 
+object syntax {
+
+  implicit final class MonadErrorOps[R[_], A](val r: R[A]) extends AnyVal {
```

Comment:
That might be useful and make the code readable in multiple places, but again I think that this could be a separate PR which not only introduces the `syntax` for new usages, but refactors old code to use it as well

Reply:
Forgot to remove that, done now

Reply:
thanks